### PR TITLE
Feat: list view

### PR DESCRIPTION
**Pull request by aeschi**

### CODE REVIEW

File: src/components/list/list.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+							{isExpanded[branchIndex] ? <MinusIcon /> : <PlusIcon />}
+						</button>
+					</div>
+					{isExpanded[branchIndex] && (
+						<div className={`flex flex-col justify-between lg:flex-row gap-9`}>
+							{ranges.map((range, rangeIndex) => (
+								<ol
+									className=" w-full flex flex-col pt-4 list-decimal list-inside lg:w-1/4"
+									key={range.name}
+								>
```
Comment (raphael-arce) [inline src/components/list/list.tsx:49]: The numbers in the design (how I understood it) should match the index order of the contact list. Also this number (the index) will be the same inside the contact points of the radar. This is necessary for print so you can match them together. If we have an ordered list for each range/branch, we can't figure out which number is which from the radar

File: src/components/list/list.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+		</div>
+	);
+};
+
+// <ul>
+// 	{items.map((item) => (
+// 		<li key={item.id}>{item.text}</li>
+// 	))}
+// 	hi
+// </ul>
```
Comment (raphael-arce) [inline src/components/list/list.tsx]: I think this can be removed 👍

File: src/components/header.tsx [hunk 3]
```diff
@@ -1,11 +1,20 @@
...
-		<header className="pt-16 pl-28">
+		<header className="pt-16 pl-28 flex flex-row justify-between">
 			<img
 				src="https://logos.citylab-berlin.org/logo-citylab-color.svg"
 				alt="CityLAB Berlin Logo"
 				className="w-48"
 			/>
+			<a
+				href="#listView"
+				className="flex text-xs flex-col items-center pr-12 gap-2"
```
Comment (raphael-arce) [inline src/components/header.tsx:13]: i'd suggest to also change in the `index.html` the `<html lang="en">`  to `<html lang="en" class="scroll-smooth">` so when you click, the jump is smooth :)👍

**Review by raphael-arce — COMMENTED** [review]
looks very nice already :) I have a couple of suggestions though 👍

File: src/components/list/list.tsx (hunk 1 above)
Comment (aeschi) [reply src/components/list/list.tsx:49]: yes, true! I will switch to the index number (+1)

**Review by raphael-arce — APPROVED** [review]
lgtm 👍

**Pull request merged**